python/cudf/cudf/_lib/cudf.pxd

```diff
@@ -31,19 +33,10 @@ cdef np_dtype_from_gdf_column(gdf_column* col)
 
 cdef get_scalar_value(gdf_scalar scalar, dtype)
 
-cdef gdf_column* column_view_from_column(col, col_name=*) except? NULL
-cdef gdf_column* column_view_from_NDArrays(
-    size,
-    data,
-    mask,
-    dtype,
-    null_count
-) except? NULL
+cdef gdf_column* column_view_from_column(Column col, col_name=*) except? NULL
 cdef gdf_scalar* gdf_scalar_from_scalar(val, dtype=*) except? NULL
-cdef gdf_column_to_column(gdf_column* c_col, int_col_name=*)
-cdef gdf_column_to_column_mem(gdf_column* input_col)
-cdef update_nvstrings_col(col, uintptr_t category_ptr)
-cdef gdf_column* column_view_from_string_column(col, col_name=*) except? NULL
+cdef Column gdf_column_to_column(gdf_column* c_col, int_col_name=*)
```

Comment:
Do we need to handle exceptions here?

Reply:
Nope. Since we're now returning a Python object exceptions should bubble up normally